vip-helpers/vip-utils.php

```diff
@@ -84,7 +87,7 @@ function vip_powered_wpcom_img_html( $image ) {
 	);
 
 	if ( array_key_exists( $image, $vip_powered_wpcom_images ) ) {
-		return '<a href="' . esc_url( vip_powered_wpcom_url() ) . '" rel="generator nofollow" class="powered-by-wpcom"><img src="' . esc_url( plugins_url( 'images/' . $vip_powered_wpcom_images[ $image ][0], __FILE__ ) ) . '" width="' . esc_attr( $vip_powered_wpcom_images[ $image ][1] ) . '" height="' . esc_attr( $vip_powered_wpcom_images[ $image ][2] ) . '" alt="' . esc_attr__( 'Powered by WordPress VIP', 'vip-helpers' ) . '" /></a>';
+		return '<a href="' . esc_url( vip_powered_wpcom_url() ) . '" rel="generator nofollow" class="powered-by-wpcom"><img src="' . esc_url( plugins_url( 'images/' . $vip_powered_wpcom_images[ $image ][0], __FILE__ ) ) . '?ver=' . rawurlencode($img_version) . '" width="' . esc_attr( $vip_powered_wpcom_images[ $image ][1] ) . '" height="' . esc_attr( $vip_powered_wpcom_images[ $image ][2] ) . '" alt="' . esc_attr__( 'Powered by WordPress VIP', 'vip-helpers' ) . '" /></a>';
```

Comment:
We encourage using the space usage in accordance with the WordPress Coding Standards:  https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage

Reply:
I knew that...updated now. Thanks for the feedback.